libimage/search.go

```diff
@@ -58,6 +58,10 @@ type SearchOptions struct {
 	InsecureSkipTLSVerify types.OptionalBool
 	// ListTags returns the search result with available tags
 	ListTags bool
+	// Registries to specify custom search registries.
```

Comment:
Document
> … if the search term does not contain a registry
? (And possibly, way to of scope, also document that the `term` parameter to `Search` actually has the [registry/]… syntax.)

(And maybe `s/search registries/registries to search`)